hl/src/H5LTanalyze.c

```diff
@@ -310,6 +310,7 @@
 #include <string.h>
 #include <errno.h>
 #include <stdlib.h>
+#include <stdint.h>
```

Comment:
Changes should be made in H5LTanalyze.l. H5LTanalyze.c is generated by lex from H5LTanalyze.l.

Reply:
The commit message mentions that `flex` generates the bits that it doesn't include for. I can include it there, but `flex` should really be adding the include for its generated code. I'll add a comment there to that effect.

Reply:
This is fixed in develop